blessings/__init__.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, kind=None, stream=None, force_styling=False):
             # send them to stdout as a fallback, since they have to go
             # somewhere.
             try:
-                setupterm(kind or environ.get('TERM', 'dumb'),
+                setupterm(kind or environ.get('TERM', 'dumb') or 'dumb',
```

Comment:
Do you see anything wrong with `setupterm(kind or environ.get('TERM') or 'dumb',`, just to remove repetition?

Reply:
Not sure if you are asking me. No, I don't see anything wrong with that. The idea is:

Use "kind", if not set, try to read environment variable "TERM" and treat an empty value like TERM is set to "dumb".

Reply:
Thanks! Tweaked and merged.